### PR TITLE
Add: containers tools into programming

### PR DESCRIPTION
**Pull request by nlamirault**
Signed-off-by: Nicolas Lamirault <nicolas.lamirault@gmail.com>

### CODE REVIEW

**Comment by mingrammer** [comment]
I think helm, krew, and kustomize should be under the `kubernetes` provider. What do you think about `diagrams.kubernetes.ecosystem` or `diagrams.kubernetes.tools`?

**Comment by nlamirault** [comment]
I prefer `diagrams.kubernetes.ecosystem`. `tools` is too generic i think. I could update the MR.

File: diagrams/programming/containers.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# This module is automatically generated by autogen.sh. DO NOT EDIT.
```
Comment (mingrammer) [inline diagrams/programming/containers.py:1]: Please delete this file manually.

Comment (nlamirault) [reply diagrams/programming/containers.py]: Done

File: docs/nodes/k8s.md [hunk 2]
```diff
@@ -5,50 +5,52 @@ title: K8S
 
 Node classes list of k8s provider.
```
Comment (mingrammer) [inline docs/nodes/k8s.md:6]: I couldn't find the `k8s.ecosystem` in this docs file. I think you should re-run the autogen.sh to update it.

Comment (nlamirault) [reply docs/nodes/k8s.md:6]: Done.

**Comment by nlamirault** [comment]
@mingrammer changelog of release v0.14.0 https://github.com/mingrammer/diagrams/releases/tag/v0.14.0 is wrong on this PR. (168 instead of 188).

Do you know when website will be available with new resources ?

**Comment by mingrammer** [comment]
@nlamirault Sorry for that :(. I updated the changelog now. And the latest documentation is now published! (k8s.ecosystem: https://diagrams.mingrammer.com/docs/nodes/k8s#k8secosystem)

**Pull request merged**